deploy/kubernetes/overlays/noauth-dev/controller-overlay.yaml

```diff
@@ -0,0 +1,34 @@
+kind: Deployment
```

Comment:
Why don't call it a noauth-debug/ rather than noauth-dev overlay. noauth is only for dev work (because it sets scope on the whole node which is too permissive).